docs/source/conf.py

```diff
@@ -228,7 +228,7 @@
 # latex_use_modindex = True
 
 # Autodoc options
-autodo_mock_imports = ["wx", "PySide", "PyQt", "PyQt5"]
+#autodoc_mock_imports = ["wx", "PySide2", "PyQt5", "PySide6", "PyQt6"]#, "pyface"]
```

Comment:
Yes, we should remove it - the previous line wasn't doing anything (note "autodo" not "autodoc") - and corrected it doesn't work because Pyface is too clever about importing.

The solution is to just not generate api docs for `traitsui.wx` and `traitsui.qt` and similar, which is sub-optimal but also literally implementation details.